core/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.35.2
```

Comment:
Please also change the version in `Cargo.toml`! :)

Reply:
```suggestion
# 0.35.2 [unreleased]
```

Reply:
Plus, I think this is a breaking change because someone using `libp2p-core` with `default-features = false` may now have a broken build if they match exhaustively on the `Keypair` enum.

Reply:
Yes, this is a breaking change, so I've bumped libp2p-core version to 0.36.0. Also, fixed tests.